### ISSUE TITLE
Question about coding in JavaScript

### ISSUE THREAD

**Issue by BoreholeSeismicLLC** [fluffynukeit/jMatFile#2]
Hi, 

I wanted to know, which file is, jMATFile.js? Also, which one is require 'fs'?

Thank you

**Comment by fluffynukeit** [comment]
I'm sorry but I don't understand your question.  Could you rephrase? This repository is a npm package.  You can download it and use it with npm like the examples show.

**Comment by BoreholeSeismicLLC** [comment]
Is there a way to run the package with javascript without installing node.js? I’m sorry, but I’m a beginner.

Appreciate all your help.

**Comment by fluffynukeit** [comment]
You might be able to but it will require modification of the code and experimentation on your part.  The javascript is in index.js.  The functions in that file are structured to use the npm package manager, and to use npm you must install Node.js.  There's no way around it that I know of.  You'd have to do this for jMatFile and also for jBinary, which is a different npm package that jMatFile uses.  I think all this will be too difficult for a beginner.  I would recommend using npm.  Why can't you install node.js?
